specification/attributes/column_naming_convention.md

```diff
@@ -24,6 +24,7 @@ Naming convention for columns appearing in billing data.
   * Column IDs SHOULD NOT use acronyms.
   * Column IDs MUST be alphanumeric with no special characters.
   * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
+  * Column IDs SHOULD NOT exceed 59 characters.
```

Comment:
nit: Can we sort the musts before the shoulds before the others?
```suggestion
  * Column IDs MUST be alphanumeric with no special characters.
  * Column IDs SHOULD NOT use acronyms.
  * Column IDs SHOULD NOT exceed 59 characters.
  * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
```

Reply:
We sure could. I don't mind doing that, we will just need a new set of reviews.

Reply:
Change done...